kube/values_test.go

```diff
@@ -0,0 +1,96 @@
+package kube
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/SUSE/fissile/model"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMakeValues(t *testing.T) {
+	t.Parallel()
+
+	assert := assert.New(t)
+	outDir, err := ioutil.TempDir("", "fissile-generate-auth-")
+	require.NoError(t, err)
+	defer os.RemoveAll(outDir)
+
+	t.Run("Check Default Registry", func(t *testing.T) {
```

Comment:
This one can probably be run in parallel? Actually, most of the other sub-cases too… That is, parallel across the various sub-tests; the toplevel test is already parallel.

Reply:
Fixed in 7a195d7